homeassistant/components/sabnzbd/__init__.py

```diff
@@ -152,6 +145,20 @@ async def async_set_queue_speed(
 
         hass.services.async_register(DOMAIN, service, method, schema=schema)
```

Comment:
You can remove this if statement as well as this function will only be called once